boards/sifive/hifive1/Kconfig.defconfig

```diff
@@ -1,12 +1,17 @@
 # SPDX-License-Identifier: Apache-2.0
 # Copyright (c) 2018 SiFive, Inc.
 
-if BOARD_HIFIVE1_FE310
+if BOARD_HIFIVE1
 
 config SYS_CLOCK_TICKS_PER_SEC
 	default 128
 
-if "$(BOARD_REVISION)" = "B"
+endif # BOARD_HIFIVE1
+
+if BOARD_HIFIVE1_REVB
+
+config SYS_CLOCK_TICKS_PER_SEC
+	default 128
```

Comment:
I think this can be factored out?

Reply:
I've decided to just leave it at the Kconfig default (i.e. remove this from defconfig), since it's not needed and caused problems in `timespec_util` tests